src/components/v5/shared/SearchSelect/partials/SearchItem/SearchItem.tsx

```diff
@@ -39,6 +64,7 @@ const SearchItem: FC<SearchItemProps> = ({
           showAvatar,
           color,
           missingPermissions,
+          isVerified,
```

Comment:
we are adding `verifiedMembers` specific things into a reusable component, can we compose that checkbox in when defining the items?